README.md

```diff
@@ -151,7 +151,7 @@ Create/Upload your code in the folder following the convention `Add Code Here`
 - Below is `an example` to add file properly
 - You can follow up `any languages`
 ```
-Add Code Here/PYTHON/<YOUR-FILERNAME>.py
+Add Code Here/PYTHON/pattern.py
```

Comment:
Why you changed here?? Its the main readme file

Reply:
I don't know how to do a task . I was learning. So, it"s by mistake